diff --git a/lib/codestar-framework/fields/image_select/image_select.php b/lib/codestar-framework/fields/image_select/image_select.php
index c3abc18..530e95e 100755
--- a/lib/codestar-framework/fields/image_select/image_select.php
+++ b/lib/codestar-framework/fields/image_select/image_select.php
@@ -43,7 +43,7 @@ if ( ! class_exists( 'CSF_Field_image_select' ) ) {
 
           echo '<div class="csf--sibling csf--image'. esc_attr( $active ) .'">';
             echo '<figure>';
-              echo '<img src="'. esc_url( $option ) .'" alt="img-'. esc_attr( $num++ ) .'" />';
+              echo '<img title="' . esc_attr( $key ) . '" src="'. esc_url( $option ) .'" alt="img-'. esc_attr( $num++ ) .'" />';
               echo '<input type="'. esc_attr( $type ) .'" name="'. esc_attr( $this->field_name( $extra ) ) .'" value="'. esc_attr( $key ) .'"'. $this->field_attributes() . esc_attr( $checked ) .'/>';
             echo '</figure>';
           echo '</div>';
